### ISSUE TITLE
Akamai Sandbox not working on M1 Mac

### ISSUE THREAD

**Issue by bjoernweide** [akamai/devpops-client#5]
When starting the sandbox (standalone or via CLI) it fails with:

```
Exception in thread "main" java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.springframework.boot.loader.MainMethodRunner.run(MainMethodRunner.java:49)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:109)
	at org.springframework.boot.loader.Launcher.launch(Launcher.java:58)
	at org.springframework.boot.loader.JarLauncher.main(JarLauncher.java:88)
Caused by: java.lang.UnsatisfiedLinkError: no conscrypt_openjdk_jni-osx-aarch_64 in java.library.path: [/Users/xxxx/Library/Java/Extensions, /Library/Java/Extensions, /Network/Library/Java/Extensions, /System/Library/Java/Extensions, /usr/lib/java, .]
	at java.base/java.lang.ClassLoader.loadLibrary(ClassLoader.java:2670)
	at java.base/java.lang.Runtime.loadLibrary0(Runtime.java:830)
	at java.base/java.lang.System.loadLibrary(System.java:1873)
	at org.conscrypt.NativeLibraryUtil.loadLibrary(NativeLibraryUtil.java:54)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
	at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:297)
	at org.conscrypt.NativeLibraryLoader$1.run(NativeLibraryLoader.java:289)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at org.conscrypt.NativeLibraryLoader.loadLibraryFromHelperClassloader(NativeLibraryLoader.java:289)
	at org.conscrypt.NativeLibraryLoader.loadLibrary(NativeLibraryLoader.java:262)
	at org.conscrypt.NativeLibraryLoader.load(NativeLibraryLoader.java:162)
	at org.conscrypt.NativeLibraryLoader.loadFirstAvailable(NativeLibraryLoader.java:106)
	at org.conscrypt.NativeCryptoJni.init(NativeCryptoJni.java:50)
	at org.conscrypt.NativeCrypto.<clinit>(NativeCrypto.java:65)
	at org.conscrypt.OpenSSLProvider.<init>(OpenSSLProvider.java:60)
	at org.conscrypt.OpenSSLProvider.<init>(OpenSSLProvider.java:53)
	at org.conscrypt.OpenSSLProvider.<init>(OpenSSLProvider.java:49)
	at com.akamai.devpops.connector.ConnectorMain.main(ConnectorMain.java:89)
```

(This might be related to https://github.com/google/conscrypt/issues/1034)

**Comment by amirilovic** [comment]
Having the same issue, any update on this one?

**Comment by bjoernweide** [comment]
I also raised it with Akamai directly, there seems to be no timeline yet. As a workaround I currently use the Sandbox client in a Docker container on top of the OpenJDK image in X68_64 compat mode.

**Comment by herzykj** [comment]
we've just released a new version of CLI-sandbox (1.6.0) which uses a new sandbox-client (1.5.0). To upgrade execute the following in the terminal:
```
akamai update sandbox
akamai sandbox install
```
After that, Sandbox should work well with Java <= 19 and Macs with M1 CPU
